### DISCUSSION TITLE
Printer resets

### DISCUSSION THREAD

**Discussion by Hellequin13** [Jyers/Marlin#1252] [Q&A] [answered]
Good morning,
I have just upgraded the firmware to Jyers, now when I attempt to preheat the printer resets itself, any suggestions please?

**Comment by kted** [comment]
Did you load the firmware default settings and saved them after you flashed?

**Comment by Hellequin13** [comment]
Hi,
I just flashed the new firmware, ran pis then tried to use the printer

Get BlueMail for Android<https://bluemail.me>
On 1 Aug 2021, at 11:20, kted ***@***.******@***.***>> wrote:

Did you load the firmware default settings and saved them after you flashed?

**Reply by Cal-LF** [reply]
The eprom should be reset after every firmware update.

**Comment by Hellequin13** [comment]
Is that the on board settings reset, or is there a specific one for eeprom, I've looked in the menu but can't find an eeprom reset.
If its the settings reset, I've done that and it still does it
Spence

Get BlueMail for Android<https://bluemail.me>
On 1 Aug 2021, at 12:59, Cal-LF ***@***.******@***.***>> wrote:

You should reset the eprom after every firmware update.

**Comment by kted** [comment, marked as answer]
Try "Reset to Defaults" from the "Control" menu
